cmd/archive_test.go

```diff
@@ -155,10 +156,10 @@ func TestArchiveNotContainsEnv(t *testing.T) {
 }
 
 // untar untars a `fileName` file to a `destination` path
-func untar(t *testing.T, fs afero.Fs, fileName string, destination string) error {
+func untar(t *testing.T, filesys afero.Fs, fileName string, destination string) error {
```

Comment:
absolutely non-blocking nit: `filesys` feels neither more accurate than `fs`, and not as explicit as `fileSystem`, the first word is not shortened.

 I don't remember where I learned this practice from, but in general, I would suggest keeping the parameter name as short as possible, or an acronym when the type itself is already super explicit: `fileSystem afero.FS` would feel kinda redundant here, and could be replaced by `f afero.FS` (although the previous `fs` would also work).

Reply:
The reason I had to rename this is that we now have a package with the `fs` name, https://pkg.go.dev/io/fs, so `fs afero.Fs` is no longer an option. Using just `f` will be too ambiguous, since I'd expect it'd represent just a single "file" or something like that. However, while `fileSystem` is a bit too long, I am fine with using it instead of `filesys` if other people also prefer it.

Reply:
Generally speaking I would prefer `filesys`, but it sounds a very good candidate for a package name, so in this case `fileSystem` could improve the readability removing ambiguity. :+1: from me for renaming.

Reply:
Maybe `fileSys` so it is still shorter but it makes it more obvious it is two words.

But I am fine with `fileSystem` as well